docker-compose.yml

```diff
@@ -76,3 +76,15 @@ services:
         depends_on:
             - metalnx-db
             - irods-catalog-provider
+
+    irods-client-nfsrods:
+        #image: irods/nfsrods:2.1.0
```

Comment:
Based on how the services are defined, people shouldn't attempt to use/enable these lines, correct?

Reply:
Right. I can remove these

Reply:
Yes. I think that's the right way to go.